tests/test_sourceTable.py

```diff
@@ -639,7 +643,8 @@ def _testFluxSlot(self, slotName):
                                  type=np.float64, doc="flux uncertainty")
         flagKey = schema.addField("%s_flag" % (baseName,),
                                   type="Flag", doc="flux flag")
-        table = lsst.afw.table.SourceTable.make(schema)
+        catalog = lsst.afw.table.SourceCatalog(schema)
+        table = catalog.table
```

Comment:
I wish someone would explain to me the difference between an Table and a Catalog...

Reply:
Roughy speaking (and I'm sure Jim or somebody else who knows better will be along to correct me in a moment):

- Tables are (confusingly named) record factories.
- Catalogs are containers of records.